test/verify/check-kdump

```diff
@@ -68,7 +70,8 @@ class TestKdump(KdumpHelpers):
         m = self.machine
 
         b.wait_timeout(120)
-        m.execute("systemctl enable kdump; systemctl start kdump || true")
+        if m.image.startswith("fedora"):
```

Comment:
let's add an `else` which asserts that the service is already running (on RHEL), to save us from surprises?